app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+      <div className="container" style={{ marginTop: '0px', display: 'flex', justifyContent: 'space-between' }}>
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
+            fontSize: '1.1rem',
+            zIndex: 1,
+            whiteSpace: 'normal',
+            width: '90%',
+            marginTop: '0px'
+
+          }}>
+
+          The emerging field of multimessenger astronomy combines the observation of disparate messenger signals to reveal unique
+          information about astrophysical sources. The four extrasolar messengers that make up multimessenger astronomy are electromagnetic
+          radiation, gravitational waves, neutrinos, and cosmic rays and the primary multimessenger sources outside the heliosphere are
+          expected to be compact binary pairs (black holes and neutron stars), supernovae, irregular neutron stars, gamma-ray bursts,
+          active galactic nuclei, and relativistic jets. The present study of the these multimessenger sources has become one of the most
+          promising discovery areas of modern astronomy. Identified as one of the central themes of the 2020 Decadal Review, the full
+          realization of time-domain and multi-messenger astronomy now stands to open an entirely new window on the Universe.
+
+          The first joint detection of two merging neutron stars in gravitational waves and electromagnetic radiation on August 17, 2017
+          was a watershed moment for the field of multimessenger astronomy. The information gained from the detection of both messengers
+          enable science that would have been impossible to perform with either signal alone. The discovery demonstrated that multimessenger
+          astrophysics offers a powerful methodology of growing importance, as astronomers combine electromagnetic radiation, gravitational
+          wave radiation, and particle astrophysics observations of cosmic events.
+
+
+          Multimessenger.nasa.gov aims to support this emerging field by consolidating important information on resources available to
+          scientists around the world to enable future discoveries.
+
+        </div>
+
+        <img
+          src={mma}
+          alt="Descriptive Alt Text"
+          style={{
+            width: 'auto',
+            height: '600px',
+            marginLeft: '0px'
+          }}
+        />
+      </div>
+
+      <div className="section-head" style={{marginTop:'150px'}}></div>
+
+      <CardGroup>
+
+        <Card
+          headerFirst
+          gridLayout={{ tablet: { col: 5 } }}
+          containerProps={{ className: '', style: { borderRadius: '10px', borderColor: '#161821' } }}
+          >
+          <CardBody className="padding-4 text-base-lightest" style={{ backgroundColor:'#161821'}}>
+            <h1 className="text-left" style={{paddingTop: '20px'}}>
+              The Decadal Survey Findings on Multi-messenger Astronomy
+            </h1>
+          </CardBody>
+        </Card>
```

Comment:
There's a lot of custom styling here.

- [ ] Please use [theme color tokens](https://designsystem.digital.gov/design-tokens/color/theme-tokens/) rather than hardcoded hexadecimal colors. Theme colors give us a consistent look and feel across the site.
- [ ] Do these custom styles define how you would like cards to appear in dark mode throughout the ACROSS site? If so, then we should move this to the [USWDS theme customization](https://designsystem.digital.gov/documentation/settings/), which probably also entails emitting two versions of the USWDS CSS, one for GCN and one for ACROSS. If not, then please factor these custom styles into a React component defined in this route.

Reply:
The closest theme color tokens to the current card background color didn't match well with the page background color (which is slightly off black), so I've left the hexadecimal colors in there for now.  I'll investigate a page background and card background combination using only the theme colors for all of the ACROSS pages and update later.